utils/profile.ts

```diff
@@ -173,58 +175,71 @@ export const getLimitedProfileTask = (
   telemetryClient: ReturnType<typeof initTelemetryClient>
 ): // eslint-disable-next-line max-params
 Task<IGetLimitedProfileResponses> =>
-  taskEither
-    .of<IGetLimitedProfileFailureResponses, void>(void 0)
-    .chainSecond(
+  pipe(
+    TE.of<IGetLimitedProfileFailureResponses, void>(void 0),
```

Comment:
We can safely remove this typings here

Reply:
Done [here](https://github.com/pagopa/io-functions-services/pull/154/commits/0dcbb5d5ddda89bbe680084525e265274f491207)